src/features/profile/Profile.tsx

```diff
@@ -1,15 +1,27 @@
-import { Nav } from "react-bootstrap";
-import { FunctionComponent } from "react";
+import React, { FunctionComponent } from "react";
+import { useSelector } from "react-redux";
+import { Nav, Navbar } from "react-bootstrap";
+import { RootState } from "../../app/store";
 
-const Profile: FunctionComponent = () => (
-  <Nav as="ul">
-    <Nav.Item as="li">
-      <Nav.Link href="/api/login" target="_blank">
-        Login
-      </Nav.Link>
-    </Nav.Item>
-  </Nav>
-);
-Profile.displayName = "Profile";
+const Profile: FunctionComponent = () => {
+
+  const profile = useSelector((state: RootState) => state.profile);
```

Comment:
Great that you're using selectors and not just passing the 'state' in.

The app is small so this is not a biggie, however, to scale better, we typically move selector functions into their own file(s) out of the view component.

```JavaScript
// profile-selectors.ts or selectors.ts etc
export const profileSelector = (state: RootState) => state.profile;
```

```JavaScript
// Profile.tsx
const profile = useSelector(profileSelector);
```

This allows the view component to decouple from the redux model.